src/agent/onefuzz/src/expand.rs

```diff
@@ -75,7 +76,8 @@ impl PlaceHolder {
             Self::SupervisorExe => "{supervisor_exe}",
             Self::SupervisorOptions => "{supervisor_options}",
             Self::SetupDir => "{setup_dir}",
-            Self::ExtraDir => "{extra_dir}",
+            Self::ExtraSetupDir => "{extra_setup_dir}",
```

Comment:
Is there a migration plan for this breaking change?

Reply:
No. To my knowledge the feature is not yet used by anyone (the person who requested it has not started using the feature).